<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaEscolhaData</class>
 <widget class="QDialog" name="telaEscolhaData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="campoData">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textobarbearia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TeInformações da barbearia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBotao">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voltarBotao">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
